packages/ble-transport/src/transports/BleInboundTransport.ts

```diff
@@ -0,0 +1,54 @@
+import type { Central } from '@animo-id/react-native-ble-didcomm'
+import type { Agent, InboundTransport, Logger } from '@aries-framework/core'
+import type { EmitterSubscription } from 'react-native'
+
+import { utils } from '@aries-framework/core'
+import { MessageReceiver } from '@aries-framework/core/build/agent/MessageReceiver'
+import { isValidJweStructure, JsonEncoder } from '@aries-framework/core/build/utils'
+
+import { BleTransportSession } from './BleTransportSession'
+
+export class BleInboundTransport implements InboundTransport {
+  public supportedSchemes: string[] = ['ble']
+  private agent!: Agent
+  private central: Central
+  private logger!: Logger
```

Comment:
We're only using these in the start method so there's no need to reassing them (`!` removes some type security)

```suggestion
  private central: Central
```